components/Ribbon/src/RibbonStyle.xaml

```diff
@@ -0,0 +1,406 @@
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="*"
+                                           MinHeight="{StaticResource RibbonGroupContentMinHeight}" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+
+                        <ContentPresenter HorizontalAlignment="Center"
+                                          VerticalAlignment="Center"
+                                          Content="{TemplateBinding Content}" />
+
+                        <TextBlock Grid.Row="1"
+                                   Style="{StaticResource RibbonGroupLabelStyle}"
+                                   Text="{TemplateBinding Label}">
+                            <ToolTipService.ToolTip>
+                                <ToolTip Content="{TemplateBinding Label}"
+                                         Placement="Bottom"
+                                         Visibility="{Binding ElementName=Label, Path=IsTextTrimmed}" />
+                            </ToolTipService.ToolTip>
+                        </TextBlock>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Key="DefaultRibbonCollapsibleGroupStyle"
+           TargetType="controls:RibbonCollapsibleGroup">
+        <Setter Property="HorizontalAlignment" Value="Center" />
+        <Setter Property="VerticalAlignment" Value="Stretch" />
+        <Setter Property="Padding" Value="{StaticResource RibbonGroupPadding}" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:RibbonCollapsibleGroup">
+                    <Grid Padding="{TemplateBinding Padding}"
+                          AutomationProperties.Name="{TemplateBinding Label}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="*"
+                                           MinHeight="{StaticResource RibbonGroupContentMinHeight}" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+
+                        <ContentControl x:Name="VisibleContentContainer"
+                                        HorizontalAlignment="Center"
+                                        VerticalAlignment="Center"
+                                        IsTabStop="False"
+                                        Visibility="Collapsed" />
+
+                        <Button x:Name="CollapsedButton"

```

Comment:
Although this might be already discussed and this is out of scope in this initial PR, might be good idea to have selected state in this button as Paint and Office apps have?

Reply:
This is indeed out of scope of this PR. It can easily be added later.